include/golioth/client.h

```diff
@@ -65,19 +65,19 @@ enum golioth_auth_type {
     GOLIOTH_TLS_AUTH_TYPE_PKI,
 };
 
-/// Pre-Shared Key (PSK) credentials.
+/// Pre-Shared Key (PSK) credential.
 ///
 /// All memory is owned by user and must persist for the lifetime
 /// of the golioth client.
-typedef struct {
+struct golioth_psk_credential {
```

Comment:
Here you have also changed the naming of the struct by dropping the `s`.

To me this is more confusing because there are multiple data in this struct. Is this change based on practices we're already using in the SDK? It might be good to mention this in the commit message.

Reply:
There is only one credential in the struct: A PSK-ID/PSK pair. The same is true for the other credential structs - each one holds a single credential.

Reply:
Keeping it as singular, but I updated the commit message for this and the others.